vortex-array/src/builders/list.rs

```diff
@@ -119,24 +119,34 @@ impl<O: OffsetPType> ArrayBuilder for ListBuilder<O> {
 
         let list = array.to_canonical()?.into_list()?;
 
-        let offset = self.value_builder.len();
-        self.value_builder.extend_from_array(list.elements())?;
+        let cursor_usize = self.value_builder.len();
+        let cursor = O::from_usize(cursor_usize).ok_or_else(|| {
```

Comment:
i realize you didn't change this here but if O::from_usize returns a result it should really be try_from_usize

Reply:
Agreed, though this is from NumTraits: https://docs.rs/num/latest/num/traits/trait.FromPrimitive.html#method.from_usize